docs/source/docs/installation/installation_prod.inc

```diff
@@ -107,9 +100,20 @@ Then in this file, set the following configuration key pair:
 This should, hopefully, make everything work. If you have problems,
 please `create an issue`_ and we’ll do our best to help.
 
-4. (Optional, but recommended) Create an apache proxy
+4. Install the JavaScript and start compiling
```

Comment:
The way we've designed it, you shouldn't have to have compile the JS or even node installed for the production build. If you `pip install megaqc` it should contain the pre-compiled JS. If this is not the case then that's a bug, but nonetheless this section shouldn't be needed in the docs. See #106 for context.